doc/_extensions/external_content.py

```diff
@@ -154,7 +154,7 @@ def sync_contents(app: Sphinx) -> None:
 
                 if not filecmp.cmp(src_adjusted, dst):
                     dst.unlink()
-                    src_adjusted.rename(dst)
+                    shutil.move(os.fspath(src_adjusted), os.fspath(dst))
```

Comment:
This is not necessary if we merge https://github.com/nrfconnect/sdk-nrf/pull/5452

Reply:
> This is not necessary if we merge #5452

I'm going to get this in now so I can build the docs until that gets merged. I gave it a +1 in the meantime.